specification/purview/resource-manager/Microsoft.Purview/preview/2024-04-01-preview/purview.json

```diff
@@ -2892,18 +2973,17 @@
       "description": "List of usage information",
       "type": "object",
       "properties": {
+        "nextLink": {
+          "description": "The Url of next link.",
+          "type": "string"
```

Comment:
Sometimes people remember to annotate these as "format": "uri". I'm not sure how much it matters, TBH.